child_compassion/views/project_compassion_view.xml

```diff
@@ -141,6 +141,7 @@
                                     <field name="time_to_medical_facility" />
                                     <field name="gps_latitude" />
                                     <field name="gps_longitude" />
+                                    <field name="google_link" widget="html" />
```

Comment:
Please make a button at the top of the form view instead of a field. It will be better accessible.

```
<button
              name="open_google_maps"
              type="object"
              string="Google Maps"
              icon="fa-external-link"
              class="oe_stat_button"
            />
```